src/autogluon_dashboard/dashboard.py

```diff
@@ -0,0 +1,12 @@
+import subprocess
+import sys
+
+from autogluon.common.loaders import load_pd
+
+if __name__ == "__main__":
+    csv1 = sys.argv[2]
+    csv2 = sys.argv[4]
+    # subprocess.run(["panel", "serve", "src/autogluon_dashboard/app.py", "--autoreload", "--args", f"{csv1}", f"{csv2}"])
```

Comment:
address in a follow-up PR, but ideally we should take in args from the user regarding whether we want to `serve` or `convert`

Reply:
Sure!